giskard/ml_worker/websocket/listener.py

```diff
@@ -584,8 +591,12 @@ def run_test_suite(
 
 
 @websocket_actor(MLWorkerAction.echo, execute_in_pool=False)
-def echo(params: websocket.EchoMsg, *args, **kwargs) -> websocket.EchoMsg:
-    return params
+def echo(params: websocket.EchoMsg, *args, **kwargs) -> websocket.EchoResponse:
+    if POOL.pool:
```

Comment:
This should be handled in utils file also, you should just call `list_in_pool` or something

Reply:
done